internal/mlflow_test.go

```diff
@@ -58,103 +69,130 @@ func TestAccMLflowRuns(t *testing.T) {
 	require.NoError(t, err)
 	t.Cleanup(func() {
 		// TODO: OpenAPI: fix mapping
-		err := w.MLflowRuns.DeleteByRunId(ctx, created.Run.Info.RunId)
+		err := w.Experiments.DeleteRun(ctx, ml.DeleteRun{
+			RunId: created.Run.Info.RunId,
+		})
 		require.NoError(t, err)
 	})
 
-	_, err = w.MLflowRuns.Update(ctx, mlflow.UpdateRun{
+	_, err = w.Experiments.UpdateRun(ctx, ml.UpdateRun{
 		RunId:  created.Run.Info.RunId,
-		Status: mlflow.UpdateRunStatusKilled,
+		Status: ml.UpdateRunStatusKilled,
 	})
 	require.NoError(t, err)
 }
 
-func TestAccRegisteredModels(t *testing.T) {
-	t.Skip("fixme")
-	t.Skip("Missing required field: name?...")
+func deleteWorkaround(t *testing.T, w *databricks.WorkspaceClient, path, key, value string) {
+	body, err := json.Marshal(map[string]string{key: value})
+	require.NoError(t, err)
+	req, err := http.NewRequest("DELETE", w.Config.Host+path, bytes.NewBuffer(body))
+	require.NoError(t, err)
+	err = w.Config.Authenticate(req)
+	require.NoError(t, err)
+	res, err := http.DefaultClient.Do(req)
+	require.NoError(t, err)
+	require.Equal(t, 200, res.StatusCode)
+}
+
+// disable once platform is fixed on mlflow side
+var deleteWorkaroundEnabled = true
+
+func TestAccModels(t *testing.T) {
 	ctx, w := workspaceTest(t)
 
-	created, err := w.RegisteredModels.Create(ctx, mlflow.CreateRegisteredModelRequest{
+	created, err := w.ModelRegistry.CreateModel(ctx, ml.CreateModelRequest{
 		Name: RandomName("go-sdk-"),
 	})
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		// TODO: OpenAPI: x-databricks-sdk-inline
-		// FIXME: Missing required field: name?..
-		err := w.RegisteredModels.DeleteByName(ctx, created.RegisteredModel.Name)
-		require.NoError(t, err)
+		deleteModel(t, w, ctx, created)
 	})
 
-	err = w.RegisteredModels.Update(ctx, mlflow.UpdateRegisteredModelRequest{
+	err = w.ModelRegistry.UpdateModel(ctx, ml.UpdateModelRequest{
 		Name:        created.RegisteredModel.Name,
 		Description: RandomName("comment "),
 	})
 	require.NoError(t, err)
 
-	_, err = w.RegisteredModels.GetByName(ctx, created.RegisteredModel.Name)
-	require.NoError(t, err)
-
-	all, err := w.RegisteredModels.ListAll(ctx, mlflow.ListRegisteredModelsRequest{})
+	all, err := w.ModelRegistry.ListModelsAll(ctx, ml.ListModelsRequest{})
 	require.NoError(t, err)
 	assert.True(t, len(all) >= 1)
 }
 
+func deleteModel(t *testing.T, w *databricks.WorkspaceClient, ctx context.Context, created *ml.CreateModelResponse) {
+	if deleteWorkaroundEnabled {
```

Comment:
What's this? Should prob track it in an issue.

Reply:
that's what we've been discussing on standup today :) we'll remove it, once MLflow service is compliant